tensorflow/core/util/gpu_solvers.h

```diff
@@ -31,13 +31,10 @@ limitations under the License.
 #include "third_party/gpus/cuda/include/cuda.h"
 #include "third_party/gpus/cuda/include/cusolverDn.h"
 #else
-#include "rocm/rocm_config.h"
 #include "rocm/include/hip/hip_complex.h"
 #include "rocm/include/rocblas.h"
 #include "tensorflow/stream_executor/blas.h"
-#if TF_ROCM_VERSION >= 40500
```

Comment:
This macro should be reinstated because hipsolver was only available starting with ROCm 4.5.

I realized that this macro was correctly implemented and that whatever issues we were seeing were caused by other changes.